src/main/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/impl/TableMapping.java

```diff
@@ -113,6 +114,15 @@ ConditionMapper getConditionMapper() {
         return dedupeFieldMappings(virtualToPhysicalMappings);
     }
 
+    @Override
+    public String toString() {
+        return format("created virtual to physical table mapping: %s -> %s, virtual: %s, physical: %s"
```

Comment:
This is an unusual `toString` function. Should the `created virtual to physical table mapping` be in a log message?

Reply:
Don't know what I was thinking on that one.  Fixed in c57167fe8f730773eb79228cbaa952f3a626f317.